### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by catalina-ramirez**
## Description

Behavior testing with jest. The tests render some components and prove if there are showing the right elements.

## Requirements

New library: @testing-library
You must run `npm install`
Also tu run the tests yo must run `npm test`

## Additional changes

Additionally I fixed the names of the page functions. They were all called Home() and I named them accordingly.

### CODE REVIEW

File: constants/forms/subproject.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ const subprojectFields = [
     validations: { required: false },
   },
   {
-    title: "Subproject Images",
+    title: "<Subproject> Images",
```
Comment (ignaciomansuy) [inline constants/forms/subproject.js:38]: porq este cambio?

Comment (catalina-ramirez) [reply constants/forms/subproject.js:38]: Se me pasó sin querer, lo arreglo!

**Review by NicoMeyer — COMMENTED** [review]
All seems fine, just remains fixing the merge conflicts.

File: .github/workflows/pr_check.yml [hunk 2]
```diff
@@ -18,8 +18,8 @@ jobs:
       - name: Run linters
         run: npm run lint
 
-      # - name: Run tests
-      #   run: npm test
+      - name: Run tests
+        run: npm test
```
Comment (FeBalla) [inline .github/workflows/pr_check.yml:22]: estás segura de que no debería ser  `npm run test`?

**Review by NicoMeyer — APPROVED** [review]
All seems fine, tests run and passed.

**Pull request merged**